media/css/quantum/_buttons.scss

```diff
@@ -20,7 +45,7 @@ a.button:visited {
         padding: .9em 40px;
```

Comment:
We should probably double check default padding for buttons too, as this special casing may no longer br required? 

Also, according to the doc that was shared in Slack (🙄), default button text should be `16px`?

Reply:
I personally think this padding looks good, and will update to 16px for the text. We'll see if Christa has more direction when she gets in.